File: src/main/webapp/app/models/Pedido/pedido-bloqueado-resumo.ts
import { TipoBloqueioEnum } from './tipo-bloqueio-enum';

export interface PedidoBloqueadoResumo {
  id?: string;
  vendedor?: string;
  cliente?: string;
  filial?: string;
  valor?: number;
  dataEmissao?: Date;
  dataEntrada?: Date;
  tiposBloqueio?: TipoBloqueioEnum[];
}

File: src/main/java/com/coremal/wconnector/app/domain/pedido/PedidoBloqueadoResumo.java
package com.coremal.wconnector.app.domain.pedido;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.util.Set;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PedidoBloqueadoResumo {

    private Long id;
    private String vendedor;
    private String cliente;
    private String filial;
    private BigDecimal valor;
    private LocalDate dataEmissao;
    private LocalDate dataEntrada;
    private Set<TipoBloqueioEnum> tiposBloqueio;
}
